### ISSUE TITLE
4.1

### ISSUE THREAD

**Issue by icyasher** [btanti/aspen-tutorials#21]
# Audio
- Venturi Scrubber and cyclone not used
- Description for CFuge updated to be related with cloth one

**Comment by btanti** [comment]
https://github.com/btanti/aspen-tutorials/commit/25248e7c45dbf96597f25a70631cdbada5660ccf

**Comment by btanti** [comment]
CFuge description still TBD

**Comment by btanti** [comment]
I took a look at the ASPEN help files -- it includes several types of centrifuges, not just the cloth one.

Solids separator: This model calculates the outlet streams from outlet stream specifications, without doing centrifuge calculations.
Perforated basket: This centrifuge separates liquids and solids by filtration using the centrifugal force of a rotating basket. The perforated basket model assumes that the separation efficiency of the solids equals 1, so that the outlet filtrate stream contains no residual solids.
Decanting centrifuge: This centrifuge consists of a rotating screw inside a drum, operating based on the principle of sedimentation.
Pusher: This centrifuge filters separates liquids and solids by filtration inside a rotating metal drum with an oscillating metal plate that pushes the solids along the drum.
Disc separator: This centrifuge forces the liquid between a stack of rotating discs which provide a large settling area. This allows it to collect very fine solid particles out of a liquid suspension

The audio we have recorded seems to  be general enough to encompass all of them,  so not sure how we  should go about changing it
